### DISCUSSION TITLE
WiFi enabled?

### DISCUSSION THREAD

**Discussion by nradina** [pschatzmann/arduino-audiokit#57] [Q&A] [answered]
Hi all,

I'm really enjoying this library! Thank you Phil for your hard work!

I've been measuring current consumption with the AudioKit development board and curious if WiFi/BT functionality is by default active with sketches that do not require Wifi or bluetooth? For example the simple Audio kit - Audio Kit sketch.

Thank you!

**Comment by pschatzmann** [comment, marked as answer]
Wifi is only active if you explicitly start it (e.g. via an URLStream or with the WiFi object)

**Reply by nradina** [reply]
Wonderful. Thank you, Phil.
